### ISSUE TITLE
[Feature Request] Graph widget

### ISSUE THREAD

**Issue by schirinowski** [martykan/forecastie#572]
I absolutely like Forecastie. What I miss is a graph widget, similar to Weather Widget showing the metrogram of yr.no (cf. https://play.google.com/store/apps/details?id=se.stephanson.WeatherWidget).
This would make me actually change completely to Forecastie.

**Comment by robinpaulson** [comment]
Yeah, that's interesting. It'd be a squeeze, but I guess modern phones have very high density screens.

**Comment by schirinowski** [comment]
Absolutely, wouldn't matter so much these days. Anyway, a single combined graph, like they show for places on www.yr.no, would do the job and won't waste vertical screen space.

The beauty of the graphs is actually that you get the content with a single glimpse without having to process all the given textual information.

**Comment by robinpaulson** [comment]
They are very thoughtfully designed, to pack a lot of information in without being crowded and still being easily readable.

**Comment by robinpaulson** [comment]
And of course on a 7 or 10 inch tablet, there's plenty of space. I often forget when working on Forecastie that Android runs on tablets as well as phones. A wide array of screen sizes.
